### DISCUSSION TITLE
Toggle to ignore info.toml name attribute

### DISCUSSION THREAD

**Discussion by RocketRaccoon27** [Raytwo/ARCropolis#456] [Ideas] [answered]
As someone who always names their folders accordingly, sometimes the name attribute in an info.toml moves mods out of "order." I don't typically read info.tomls so this leads me to having to scroll through all of my mods list to find a mod I don't know the name of. If I do see an info.toml ahead of time, I usually delete it but this leads to a loss of information (namely the category function). To alleviate this, I propose a toggle for ARCropolis that would ignore the name attribute and use the folder name.

**Comment by Raytwo** [comment, marked as answer]
Added to ARCropolis 3.11.0
